pygfunction/examples/custom_borehole.py

```diff
@@ -141,13 +142,29 @@ def main():
     Rb_star_UBH = Rb * eta * (1 / np.tanh(eta))  # eq. (3.68)
 
     Rb_ = Coaxial._Rd[1, 1]  # Local borehole thermal resistance
+    print('Local borehole resistance: {}'.format(Rb_))
     assert Rb_ == Rb
     assert R_ff == Ra
 
+    print(R_conv_1)
+
+    def func(_Rd_i):
+        Coaxial._Rd[0, 0] = _Rd_i
+        Coaxial._initialize_stored_coefficients()
+        Rb_calc = gt.pipes.borehole_thermal_resistance(Coaxial, m_flow_borehole, fluid.cp)
+        return Rb_star_UBH - Rb_calc
```

Comment:
I decided that `np.inf` for that value looked a bit peculiar. So I found the value of `Coaxial._Rd[0, 0]` that drove the `func`s value to its root.

Reply:
Nevermind, that's a coincidence.